mtp_cashbook/templates/base.html

```diff
@@ -32,11 +35,20 @@
   </div>
   {% endif %}
 
+  <div class="subheader-item phase-banner">
+    <span class="phase-tag phase-tag-alpha">
+      ALPHA
+    </span>
+    <span class="phase-banner-message">
+      This is a new service – your <a href="#">feedback</a> will help us to improve it.
```

Comment:
Might be worth using `#TODO` instead of a raw `#` to make these easier to find? Also, maybe wrap all of `your feedback`?